modelling_earth/io.py

```diff
@@ -326,3 +326,34 @@ def read_swarm(path):
     # Create a dictionary
     swarm = {"time": time, "step": steps, "positions": positions}
     return swarm
+
+
+def save_swarm(swarm, name, save_path):

```

Comment:
I would add a `**kwargs` argument to be passed to `to_hdf()` method. Moreover, the arguments for `to_hdf()` must all be set inside the `kwargs` dictionary. I can help you doing this if you need to.